i18n/en-US.properties

```diff
@@ -108,6 +108,8 @@ be.drawAnnotation = Drawing annotation mode
 be.editLabel = Edit
 # Message to the user when there are no Open With integrations
 be.emptyOpenWithDescription = No integrations are available for this file
+# Generic error label.
+be.error = Error
```

Comment:
We already have this exact string in 2 other places, should we remove those and keep this generic one?

Reply:
I'll do a followup removing the other versions